.vale/styles/RedHat/DoNotUseTerms.yml

```diff
@@ -17,6 +17,7 @@ swap:
   devs?: 'Do not use "dev" as a synonym for development or developer.'
   foo: 'Do not use "foo". This term is technical jargon in code and as shorthand for fubar, an acronym of profanity in code.'
   fubar: 'Do not use "fubar". This term is an acronym of a profanity that is sometimes used by developers in code.'
+  green|yellow|red|blue: 'Do not use visual language, like color, to describe items in the us unless it is also described non-visually, like with the name of the item.' 
```

Comment:
Not sure about this as a "Do Not Use" term. For one thing, this will throw an error at every instance of "Red Hat" in source. We've also had strong feedback about noise in the Vale rules and we've had some teams actively turn away from our Vale rules because of this. My inclination is to leave out an error term if there is ambiguity as to its correctness in a given context.  

ISG says: "[Don't] Use color as the only way to convey meaning; information that is conveyed by color should also be evident when it's rendered without color."

This does not say that we shouldn't use colors in our written content, just that we shouldn't use it as the _only_ way to convey meaning.

https://www.ibm.com/docs/en/ibm-style?topic=medium-accessibility

Reply:
Ok, finally got around to this. I removed the new rule that I was trying to make called visual language, and put all the  items into already existing rules. I also got rid of the admonition against Red.